### PR TITLE
Create LIT5843ShortPoemMary.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record

### CODE REVIEW

File: new/LIT5843ShortPoemMary.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2020-03-04">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT5843ShortPoemMary" passive=""/>
```
Comment (eu-genia) [inline new/LIT5843ShortPoemMary.xml]: passive value?

File: new/LIT5843ShortPoemMary.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                <language ident="am">Amharic</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2020-03-04">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
```
Comment (eu-genia) [inline new/LIT5843ShortPoemMary.xml]: the text is not identifiable without at least an incipit; how can we know that this is not one of the many poems already present?

**Comment by thea-m** [comment]
@eu-genia can this be merged?

**Comment by eu-genia** [comment]
Sorry I have preapproved but I believe the corrections I had asked for are not there, or I cannot see them (text and passive value in the relation in LIT5843ShortPoemMary), please correct before merging

**Comment by DenisNosnitsin1970** [comment]
Done. I can get to the text only when I am in the office.

**Pull request merged**